en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/api-key-authentication.md

```diff
@@ -0,0 +1,142 @@
+# API Key Authentication
+
+API Key Authentication in WSO2 Choreo Connect is a simple authentication scheme that accepts a valid self-contained JWT token issued for accessing APIs. Since self-contained tokens can have all authorization information, the token processing time is fast and more efficient. Common practice is to have a short expiration time for self-contained access tokens.
+
+Follow the instructions below to use API Key Authentication with WSO2 Choreo Connect:
+
+## Step 1 - Define API Key Authentication for the API
+
+Use one of the following two methods to enable API Key Authentication in Choreo Connect.
+
+### Method 1 - Using OpenAPI definition security schemes
+
+Follow the instructions below to enable API Key Authentication using OpenAPI definition security schemes.
+
+-  **Step 1 - Define the API Key security scheme**
+
+     Security schemes must be defined on the OpenAPI definition under `securitySchemes`. One or more API key security schemes can be used (as in logical OR) simultaneously. Define the API Key security scheme as follows:
```

Comment:
@slahirucd7 , there is content missing within the brackets. Please let me know.

Reply:
Applied the same explanation in OAS3 documentation. The intention was to say that users can apply multiple API Key security schemes for a given API resource.

![image](https://user-images.githubusercontent.com/40932779/139820316-374b1b45-470e-400f-82e6-6f7becd56b25.png)

Reply:
Updated as follows:

"One or more API Key security schemes can be used simultaneously, similar to a logical OR operator."


https://github.com/wso2/docs-apim/pull/4958/commits/258fd98ad29e1d9e79e4301ec99abb97c1d246d8